services/ingestion/etl/2014tc16rfcb047/xls/test/unit/lib/__snapshots__/transform.spec.js.snap

```diff
@@ -0,0 +1,98 @@
+      "currency": "EUR",
+      "raw": 8506928.97,
+      "value": 8506928.97,
+    },
+  },
+  "call_year": "",
+  "complete": false,
+  "description": "Theme: 1.1
+eMS Ref Num: 5031
+Operation/Project Summary: The Border and Regions Airways Training Hub (BREATH) Project, led by the Dundalk Institute of Technology (DIT), has been awarded up to €6,781,065.21 ERDF under the INTERREG VA Programme. BREATH aims to tackle Chronic Obstructive Pulmonary Disease (COPD) by harnessing the complementary resources and expertise of the project partners. COPD is an incurable lung disease that is highly prevalent in Northern Ireland, the Border Counties of Ireland, and Western Scotland. COPD imposes a huge economic burden on the Region and it has been forecast that the disease will be responsible for one-third of deaths by 2020. BREATH will provide an innovative, industry-relevant training programme to stimulate R&I, attract inward investment and enhance economic development.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               Project Partners: Queen's University Belfast and the University of the West of Scotland. 
+Committed Outputs: Enterprises receiving support: 5
+Enterprises receiving grants: 2
+Enterprises receiving non-financial support: 5
+Researchers in supported entities: 22 (total of 89.5 researcher years)
+Enterprises cooperating with research institutions: 5                                                                                                                                                                                                                                             Enterprises participating in cross-border, transnational or interregional research projects: 2                                                                                                                                                    Research institutions participating in cross-border, transnational or interregional research projects: 3
+Total ERDF + Match (€): 7727271.2
+Total ERDF Allocated (€): 6781065.21
+Category of Intervention: 060
+",
+  "ec_priorities": Array [],
+  "media": Array [],
+  "programme_name": "",
+  "project_id": "045",
+  "project_locations": Array [
+    Object {
+      "address": "Northern Ireland, Ireland & Scotland",
+      "centroid": null,
+      "country_code": "",
+      "location": null,
+      "nuts": Array [],
+      "postal_code": "A91 K584",
```

Comment:
seems like `A91 K584` is in Dundalk, Ireland, yet the Excel file says `Northern Ireland, Ireland & Scotland`. I don't know what's the best way to extract the `project_locations`. Has Laurent said anything about it?

Reply:
You're right, I've assumed too much to put countries into address and hope for post code to help.
I changed the logic to use the country names as much as possible to deduct country codes and remove the post code. In this particular example, Scotland and Northern Ireland are not deducted, but Ireland is, so we store its country code which is more precise than the assuming-address with the postcode.